{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -48,6 +48,20 @@
     },
 }
 
+
+REDIS_URL = env("REDIS_TLS_URL", default=env("REDIS_URL"))
+REDIS_SSL = env.bool("REDIS_SSL", default=False)
+
+CERT_NONE = 0
+
+if REDIS_SSL:
+    CELERY_REDIS_BACKEND_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
+    CELERY_BROKER_USE_SSL = {"ssl_cert_reqs": CERT_NONE}
+    CACHES["default"]["OPTIONS"]["CONNECTION_POOL_CLASS"] = "redis.connection.SSLConnection"
+    CACHES["default"]["OPTIONS"]["SSL_CERT_REQS"] = CERT_NONE
+
+CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
```

Comment:
That's out of scope I think, please revert, see https://github.com/cookiecutter/cookiecutter-django/issues/5085

Reply:
Yes, the CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP setting is better covered in issue #5085, I will remove it from this PR.